src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml

```diff
@@ -9,7 +9,6 @@
              xmlns:resx="clr-namespace:Dynamo.Properties;assembly=DynamoCore"
              xmlns:viewmodels="clr-namespace:Dynamo.ViewModels;assembly=DynamoCoreWpf"
              mc:Ignorable="d"
-             Width="250"
```

Comment:
This is minor (because menu is not likely to get too small), but do we need to have a `MinWidth` attribute? What if we choose to remove lengthy menu items and caused the menu to go below `250`? Might not be relevant now I guess...